src/test/java/api/support/http/ResourceClient.java

```diff
@@ -174,6 +174,11 @@ public static ResourceClient forPatronNotices(OkapiHttpClient client) {
       "patron notice", "patronnotices");
   }
 
+  public static ResourceClient forScheduledNotices(OkapiHttpClient client) {
+    return new ResourceClient(client, InterfaceUrls::scheduledNoticesUrl,
+      "scheduled notice", "schedulednotices");
```

Comment:
If the array property name is capitalised (see above), this will also need to change

Reply:
Done